src/components/atoms/SelectBox/SelectBox.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+import { ChangeEvent } from 'react';
+
+type SelectBoxProps = {
+  fontSize?: string;
+  borderRadius?: string;
+  selectAll?: boolean; // true 전체선택, 값이 없으면 or false 선택하세요 표시
+  id?: string;
+  name?: string;
+  disabled?: boolean; // true시 disabled처리 값이 없거나 false일때는 사용가능
+  options: Record<string, string | number>; //예시 => const option = { 남자: 1, 여자: 2 };
+  onChange: (event: ChangeEvent<HTMLSelectElement>) => void; //예시 const testChange = (event: ChangeEvent<HTMLSelectElement>) => { console.log(event.target.value);};
+};
+
+const SelectBox = ({
+  fontSize,
+  borderRadius,
+  selectAll,
+  id,
+  name,
+  disabled,
+  options,
+  onChange,
+}: SelectBoxProps) => {
+  return (
+    <SelectBoxWrap>
+      <SelectBoxBox
+        fontSize={fontSize}
+        borderRadius={borderRadius}
+        disabled={disabled}
+        id={id}
+        name={name}
+        onChange={onChange}
+      >
+        {selectAll ? (
+          <option value="99">전체선택</option>
+        ) : (
+          <option value="0">선택해주세요</option>
+        )}
+        {Object.entries(options).map(([option, value]) => (
+          <option key={value} value={value}>
+            {option}
+          </option>
+        ))}
```

Comment:
제가 지난번에 모호하게 코멘트를 했던 것 같네요.
options라는 object의 key를 option의 식별자로 사용하고 (option tag의 key와 value)
object의 value를 option tag 안에 표시되는 내용으로 적는 것이 어떤지 의견 드렸던 거였습니다.
아래 코드 예시 참고해주세요
```suggestion
        {Object.entries(options).map(([key, value]) => (
          <option key={key} value={key}>
            {value}
          </option>
        ))}
```